src/p_pspr.c

```diff
@@ -53,11 +53,11 @@ static struct
   { 10, 2 },   // wp_pistol
   { 30, 4 },   // wp_shotgun
   { 10, 2 },   // wp_chaingun
-  { 100, 8 }, // wp_missile
+  { 100, 7 }, // wp_missile
   { 20, 2 },   // wp_plasma
-  { 100, 10 }, // wp_bfg
+  { 100, 9 }, // wp_bfg
   { 0, -1 },   // wp_chainsaw
-  { 80, 8 }   // wp_supershotgun
+  { 80, 7 }   // wp_supershotgun
```

Comment:
The halved values were already very close:

| description            | original height | rfomin angle | exact angle ceiling | exact angle |
|------------------------|-----------------|--------------|---------------------|-------------|
| recoil wp_fist         |               0 |            0 |                   0 |       0.000 |
| recoil wp_pistol       |               4 |            2 |                   2 |       1.718 |
| recoil wp_shotgun      |               8 |            4 |                   4 |       3.434 |
| recoil wp_chaingun     |               4 |            2 |                   2 |       1.718 |
| recoil wp_missile      |              16 |            8 |                   7 |       6.843 |
| recoil wp_plasma       |               4 |            2 |                   2 |       1.718 |
| recoil wp_bfg          |              20 |           10 |                   9 |       8.531 |
| recoil wp_chainsaw     |              -2 |           -1 |                  -1 |      -0.859 |
| recoil wp_supershotgun |              16 |            8 |                   7 |       6.843 |
| recoil centering rate  |               1 |            1 |                   1 |       0.430 |
| view centering rate    |               8 |            8 |                   4 |       3.434 |

Height to angle formula:
```c
angle = atan((height * 1.2) / 160.0) * (180.0 / M_PI)
```